P0009/P0009.md

```diff
@@ -37,11 +37,51 @@ toc: true
 
 ## P0009r14: 2021-11 Mailing
 
-- made `extents` converting constructor conditionally explicit, for cases where dynamic extents are turned into static extents
+#### LEWG Review 11/01/2021
+- ACTION: Maybe a default constructible accessor should imply default constructible spans.  Default constructible spans are important in many cases.
+  - done
+- ACTION: Make extents conditionally explicit when converting from dynamic to static extents.
+  - done
+- ACTION: Make layout and mdspan converting constructors conditionally explicit based upon the underlying types
+  - done
+- ACTION: constructing extents from std::array needs a deduction guide?
+  - not done: you can't do this since you can't use alias and you can't return a parameter pack
+- QUESTION: Extents parameter pack ctor needs to be explicit?  Deduction guide needs to be explicit?
+  - we made that change
+- QUESTION: submdspan constructor can take tuple<size_t, size_t> and get pair<size_t, size_t> for free (could add specific type for submdspan, so long as it can be converted from pair and tuple).
+  - we changed submdspan to take things convertible to `tuple<size_t,size_t>`
+- ACTION: needs feature test macro
+  - done
+- QUESTION: Can get rid of mdspan ctor takes pointer + array?  Can construct extent from array.  Disagreement on authors on this one.
+  - not done: would remove ctad from array
+- ACTION: Explore modifying the requirements on which extents need to be provided when constructing an mdspan or extents object.
+  - done
+  - Authors decided to enable construction from both dynamic extents only, and all extents (for both integer packs and arrays)
+  - Why from dynamic extents only:
+    - no redundant information
+    - precondition free constructor
+    - enables fully static extents mdspan construction from ptr only
+  - Why from all extents:
+    - no confusion what extent a given argument is associated with
+    - enables easier writing of certain types of generic code e.g.:
+      ```c++
```

Comment:
Would you consider perhaps the following small change to the example, so that LEWG won't waste time fussing about the dangling pointer?

      ```c++
      constexpr std::size_t storageSize = 1000000;
      std::array<double, storageSize> storage;
            
      template<class mds1_t, class mds2_t>
      auto gemm_result(mds1_t mdspan1, mds2_t mdspan2) {
         using return_t = mdspan<double,Extents<mds1_t::extents_type::static_extent<0>,mds2_t::extents_type::static_extent<1>>;
         assert(mdspan1.extent(0)*mdspan2.extent(1) <= storageSize);
         return return_t(storage.data(), mdspan1.extent(0),mdspan2.extent(1));     
      ```